training/src/main/scala/com/airbnb/aerosolve/training/TrainingUtils.scala

```diff
@@ -179,6 +203,16 @@ object TrainingUtils {
     }
   }
 
+  def getLabel(example: Example, loss: String, rankKey: String, threshold: Double): Int = {
+    val fv = example.getExample.get(0)
+    val label: Int = if (loss == "regression") {
+      TrainingUtils.getLabel(fv, rankKey).toInt
```

Comment:
but the loss is regression, the label should be double. If this is returning classification label, perhaps we should not allow the loss to be 'regression'